docker/images/brownie/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9
+FROM --platform linux/amd64 python:3.9

```

Comment:
Based on the docs [1] this line needs to be:

FROM --platform**=**linux/amd64 python:3.9

[1]: https://docs.docker.com/engine/reference/builder/#from